packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -27,11 +27,14 @@ import {
 import dialogPolyfill from 'dialog-polyfill';
 @customElement('opc-feedback')
 export class OpcFeedback extends LitElement {
-  @property({ type: String, attribute: 'spa' }) spa = '/feedback';
-  @property({ type: String, attribute: 'docs' }) docs = '/get-started';
+  @property({ type: String, attribute: 'spa' }) spa = undefined;
+  @property({ type: String, attribute: 'docs' }) docs = undefined;
```

Comment:
@yashoswalyo why were the defaults removed?

Reply:
To hide it from UI if someone has no SPA for feedback/docs

Reply:
Please check https://lit.dev/docs/templates/directives/#ifdefined

Reply:
I tried with `ifDefined()`, it renders the element and removes href attribute when `undefined`. 
Which is not the expected outcome.